cli/src/error.rs

```diff
@@ -57,4 +57,7 @@ pub enum Error {
 
 	#[error("This subcommand is only available when compiled with `{feature}`")]
 	FeatureNotEnabled { feature: &'static str },
+
+	#[error("Insecure validator: {0} Run with --insecure-validator-i-know-what-i-do if you understand and accept the risks of running insecurely.")]
```

Comment:
"Insecure validator" can be kinda vague, maybe something like this?

```suggestion
	#[error("Your system cannot securely run a validator: {0}\nYou can ignore this requirement by using the `--insecure-validator-i-know-what-i-do` command line argument if you understand and accept the risks of running insecurely.")]
```

Also, I think in this case it'd be nice to maybe e.g. put a link to the docs with a longer explanation of *why* this is and what will be the consequences when running in unsecure mode.

Reply:
Like the idea, just don't know where the doc should go. Maybe on the validators guide?